src/App.tsx

```diff
@@ -1,24 +1,56 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useState } from 'react';
 
-const USER_ID = 0;
+import { TodosHeader } from './Components/TodosHeader';
+import { TodosFooter } from './Components/TodosFooter';
+import { Todoslist } from './Components/TodosList';
+import { UserWarning } from './UserWarning';
+import { ErrorNotification } from './Components/ErrorNotification';
+import { ErrorMessage } from './Enum/ErrorMessage';
+import { getTodos } from './api/todos';
+import { useTodo } from './Hooks/UseTodo';
+import { USER_ID } from './variables/userId';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const { setTodos, setIsError, setLoading } = useTodo();
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [errorVisibility, setErrorVisibility] = useState(false);
+  const [processings] = useState<number[]>([]);
```

Comment:
```suggestion
  const [processings, setProcessings] = useState<number[]>([]);
```
do you really not need setProcessings function for update processings state?
looks like this state can be used for manage todos which in some process on server (updating, deleting)

Reply:
I planned to add it later